dotnet-desktop-guide/net/winforms/printing/snippets/how-to-print-in-windows-forms-using-print-preview/vb/Form1.vb

```diff
@@ -0,0 +1,73 @@
+﻿Imports System.Drawing.Printing
+Imports System.IO
+Public Class Form1
+    '<string_declaration>
+    ' Declare a string to hold the entire document contents.
+    Private DocumentContents As String
+    ' Declare a variable to hold the portion of the document that
+    ' is not printed.
+    Private StringToPrint As String
```

Comment:
These shouldn't be title case. The name should be `documentContents` and `stringToPrint`